README.md

```diff
@@ -1,23 +1,57 @@
 # Testsuite for the Dyninst tool for binary instrumentation, analysis, and modification
 
-## Usage
+## Building Testsuite
 
-Because Dyninst and its testsuite are tightly integrated, it is highly recommended to use the build script located in `scripts/build/build.pl`.
+To build testsuite, please first have the Dyninst version you want to test installed.
+Then one can build the testsuite using cmake by specifying the path of dyninst installation using Dyninst\_ROOT.
 
-Example usage on Linux:
+```
+mkdir build
 
-	> export PERL5LIB=testsuite/scripts/build
-	> perl testsuite/scripts/build/build.pl --njobs=4
-	
-The build script has several options for configuring library locations. See `build.pl --help` for details.
+cd build
 
-### Running tests for pull requests
+cmake .. -DDyninst_ROOT=/path/to/your/dyninst/installation -DCMAKE_INSTALL_PREFIX=/path/to/your/testsuite/intallation
```

Comment:
```suggestion
cmake .. -DDyninst_ROOT=/path/to/your/dyninst/installation -DCMAKE_INSTALL_PREFIX=/path/to/your/testsuite/intallation
cmake --build .
cmake --install .
```

I'm not entirely sure why, but the testsuite can't be run from its build directory. It has to be installed first.